tools/bin/commands/minikube

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+
+minikube::description() {
+    echo "Provides helpful minikube functions"
+}
+
+minikube::usage() {
+    cat <<"EOT"
+volumes                      Create volumes for syndesis installation
+start                        Start a new instance of minikube
+      Start Switches:
+       --reset                   Remove any old instance of minikube before starting
+       --driver                  The driver to use for accessing the VM (kvm2 by default)
+       --disk                    The size of disk to allocated in GB (60GB by default)
+       --memory                  The size of memory to allocate GB (12GB by default)
+       --cpus                    The number of CPUs to allocate (4 by default)
+       --registeries             A file listing any registeries the cluster should have access to
```

Comment:
A typo, should be **registries**

Reply:
Well I was at least consistently spelling it wrong :-)